src/spacesettlers/simulator/Toroidal2DPhysics.java

```diff
@@ -985,25 +1017,9 @@ private double checkAngularVelocity(double angularVelocity) {
 		}
 	}
 
```

Comment:
Did this checkTranslationalVelocity go away completely?

Reply:
Yup